youtube_dl/extractor/bloomberg.py

```diff
@@ -48,13 +48,20 @@ def _real_extract(self, url):
             r'["\']bmmrId["\']\s*:\s*(["\'])(?P<url>.+?)\1',
             webpage, 'id', group='url', default=None)
         if not video_id:
-            bplayer_data = self._parse_json(self._search_regex(
-                r'BPlayer\(null,\s*({[^;]+})\);', webpage, 'id'), name)
+            bplayer_json = self._search_regex(r'BPlayer\(null,\s*({[^;]+})\);',
+                webpage, 'id')
+            # It's not good JSON; it uses single quotes and contains function
```

Comment:
Unfortunately, that doesn't work. I get `"bb_region": "getCookie"("ak_rg")` in the "JSON".